### PR TITLE
4.7:Add Quadplane QwikTune

### PR DESCRIPTION
**Pull request by Hwurzburg**
@MichelleRos I created shell for the QwikTune PR and highlighted in bold sections that need your input...if you could squash your added commits to this to a single commit, it would save me a step later...thanks

### CODE REVIEW

**Review by MichelleRos — CHANGES_REQUESTED** [review]
This seems to be adding a new page called "qwiktune" to describe the usage of the quicktune c++ in Plane. I don't think this is the right way to go. We don't want to be maintaining two separate pages.

What I was suggesting on Discord was that we make the current quicktune page in Copter be available under Plane.

**Comment by Hwurzburg** [comment]
Copter uses the LUA script not the C++ code...changing the C++ code does not impact the Copter page and vice versa...IF Copter ever adopts QwikTune then it might be worthwhile...

however, if you want to create a common page (common-quicktune.rst) the differences in text (which will be significant as will the other link changes throughout two wiki's tuning sections) can be isolated to a particular wiki with [site wiki="plane"] [/site]  or [site wiki="copter"] [/site] delimiters around wiki site specific wording, params, links, etc.

**Comment by Hwurzburg** [comment]
@MichelleRos since this is back ported to 4.6, there is some urgency to complete and merge this.....can you contribute?

**Comment by MichelleRos** [comment]
@Hwurzburg The intent AFAIK was to keep the Lua and C++ implementations the same (well, actually the intent was to get rid of the Lua implementation and just have the C++ implementation for both copter and plane instead of doubling our maintenance workload, but certain people refused to allow that).

As for the documentation, I don't really know whether there would be enough differences between the two to make it easier to do two separate pages, but that'd really be a question for @tridge as quicktune is really his thing.

**Comment by Hwurzburg** [comment]
I have updated and will merge once Tridge approves

File: plane/source/docs/quicktune.rst
Comment (tridge) [inline plane/source/docs/quicktune.rst:39]: QLOITER not Loiter

File: plane/source/docs/quicktune.rst
Comment (tridge) [inline plane/source/docs/quicktune.rst:42]: it does not restore the gains while repositioning

File: plane/source/docs/quicktune.rst
Comment (tridge) [inline plane/source/docs/quicktune.rst:43]: the tune will automatically abort if an angle error over QWIK_ANGLE_MAX is detected

**Review by tridge — CHANGES_REQUESTED** [review]

**Pull request merged**
